File: Php/app/Urls/CashgameEditUrlModel.php
<?php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\CashgameUrlModel;
	use entities\Cashgame;
	use entities\Homegame;

	class CashgameEditUrlModel extends CashgameUrlModel{

		public function __construct(Homegame $homegame, Cashgame $cashgame){
			parent::__construct(RouteFormats::cashgameEdit, $homegame, $cashgame);
		}

	}

}

File: Web/Models/UrlModels/CashgameEditUrlModel.cs
using Core.Classes;
using Web.Routing;

namespace Web.Models.UrlModels{

	public class CashgameEditUrlModel : CashgameUrlModel{

		public CashgameEditUrlModel(Homegame homegame, Cashgame cashgame)
            : base(RouteFormats.CashgameEdit, homegame, cashgame)
        {
		}

	}

}